enzyme/Enzyme/AdjointGenerator.h

```diff
@@ -979,15 +983,6 @@ class AdjointGenerator
 
   void visitBinaryOperator(llvm::BinaryOperator &BO) {
     eraseIfUnused(BO);
-    if (gutils->isConstantInstruction(&BO))
```

Comment:
I think you'll still want the activity checks for fwd mode.

Reply:
To avoid inadvertently differentiating a +[integer] b

Reply:
Activity checks are on line 1414 and 1418

Reply:
Yes, for the operands of the individiual operator. It may still be useful to do an initial check of the entire instruction is inactive (and thus you don't need to run createBinaryOperatorDual /  createBinaryOperatorAdjoint at all)